test/e2e/storage/external/external.go

```diff
@@ -120,10 +121,6 @@ type driverDefinition struct {
 		ReadOnly bool
 	}
 
-	// SupportedSizeRange defines the desired size of dynamically
-	// provisioned volumes.
-	SupportedSizeRange e2evolume.SizeRange
```

Comment:
my understanding is that setting this field never worked right? So it should be safe to remove without breaking anyone.

Reply:
Yes. This is never working. Nobody is consuming this field.